src/PerformanceTests/Tests/Categories/AzureFixture.cs

```diff
@@ -26,10 +18,13 @@ static IEnumerable<Permutation> CreatePermutations()
         {
             return PermutationGenerator.Generate(new Permutations
             {
-                Transports = new[] { Transport.AzureStorageQueues, Transport.AzureServiceBus,  },
-                Persisters = new [] { Persistence.Azure, },
-                Serializers = new[] { Serialization.Json, },
-                OutboxModes = new[] { Outbox.Off, },
+                Versions = new[] { NServiceBusVersion.V7 },
+                Transports = new[] { Transport.MSMQ },
```

Comment:
why MSMQ?

Reply:
@timbussmann Because MSMQ is the least impacting any persistence performance-related tests.

Reply:
Looking at the acceptance tests it definitely seems like the learning transport is faster than MSMQ, would be interesting to see if that would hold true for the perftests as well?